lenskit-core/src/main/java/org/lenskit/util/ProgressLogger.java

```diff
@@ -133,7 +133,7 @@ public void logProgress() {
             double est = (total - ndone) * rate;
             logger.info("{}: finished {} of {} ({}%, {}s/row, ETA {})",
                         label, ndone, total,
-                        String.format("%.2f", ((double) ndone) / total),
+                        String.format("%.2f", (((double) ndone) * 100) / total),
```

Comment:
We can simplify this, though: multiplying `ndone` by `100.0` (note the decimal point) will convert `ndone` to a double :).